website/server/models/group.js

```diff
@@ -609,7 +609,31 @@ schema.methods.sendChat = function sendChat (message, user, metaData) {
   };
 
   User.update(query, lastSeenUpdateRemoveOld, {multi: true}).exec().then(() => {
-    User.update(query, lastSeenUpdateAddNew, {multi: true}).exec();
+    User.find(query).exec(async (err, users) => {
```

Comment:
I don't think that this will solve the problem completely because not having transactions the problem will always remain unless we can fit the check in the update query... anyway fetching the user and saving it is too slow to do it every time a notification is added to an user